docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -116,6 +116,12 @@ values. Typical usage does not require you to configure them.
       - Enables :ref:`Table statistics <delta-lake-table-statistics>` for
         performance improvements.
       - ``true``
+    * - ``delta.rename-managed-tables.enabled``
+      - Enable :ref:`renaming tables <renaming-tables>` created without
+        explicit location. A rename moves the data on the underlying storage
+        system. As moving data on the underlying storage system is a potentially
```

Comment:
apologies, but i am still unsure what the rename semantics for tables with implicit locations should be.
there is a bunch of options, and sending rename to metastore is one of them, but i am not yet convinced it's the best one.